app/models/application_setting.rb

```diff
@@ -0,0 +1,14 @@
+# == Schema Information
```

Comment:
Add an enum for creating a table dropdown. It would prevent mis entering a table name. Also adding a validation for uniqueness on the page field would prevent collisions